docs/getting-started.md

```diff
@@ -0,0 +1,105 @@
+# Getting Started - BAG Bevragen API
+De 'BAG Bevragen' API levert actuele gegevens uit de Basisregistratie Adressen en Gebouwen. Dit betekent dat geen historische gegevens kunnen worden gevraagd en dat gegevens met een eindstatus niet worden geleverd.
+
+De API is gespecificeerd met behulp van de OpenAPI Specifications (OAS3).
+
+Om aan te sluiten kun je de volgende stappen doorlopen:
+1. [Meld je aan bij het kadaster om toegang te krijgen](#Aanmelden-om-aan-te-sluiten)
+2. [Bekijk de functionaliteit en specificaties](#Functionaliteit)
+3. [Probeer en test de API](#Probeer-en-test-de-API)
+
+## Aanmelden om aan te sluiten
+Vraag [een API-key aan](https://formulieren.kadaster.nl/aanvragen_bag_api_key_testomgeving) om toegang te krijgen tot de BAG API.
+
+## Functionaliteit
+U kunt de Open API Specificaties (OAS3) van de API bekijken in [Swagger-formaat](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BAG-Bevragen/master/api-specificatie/openapi.yaml).
+
+De (resolved) OAS3 is hier te downloaden: [openapi.yaml](../api-specificatie/resolved/openapi.yaml).
+
+### Beschikbare resources
+De API kent de volgende resources:
+- adressen: hierin zijn samenhangende en gerelateerde gegevens samengevoegd uit de nummeraanduiding, openbare ruimte en woonplaats die samen een adres vormen
+- adresseerbareobjecten: dit kan een verblijfsobject, ligplaats of standplaats zijn
+- nummeraanduidingen
+- openbareruimten
+- woonplaatsen
+- panden
+
+Voor de verschillende resources wordt de uri samengesteld met de identificatie (lokaal id) van het onderliggende BAG-object. Dit is een 16 cijferige, of in geval van woonplaats 4 cijferige identificatie. Voor adressen wordt de nummeraanduiding identificatie gebruikt in de uri.  
+De identificatie kan voorloopnullen bevatten, dus het is geen integer.
+
+#### Zoeken van een adres
+Een adres kan gezocht worden met parameter "zoek" waarin met postcode, woonplaats, straatnaam, huisnummer, huisletter en huisnummertoevoeging kan worden gezocht. Op dit moment worden alleen deze gegevens ondersteund, het is de bedoeling dat dit later volledig "fuzzy search" wordt.  
+Deze zoekfunctie, via endpoint /adressen/zoek levert voor elk gevonden adres een combinatie van een zoekresultaat identificatie en een omschrijving, plus een link voor het ophalen van de volledige adresgegevens.
+
+Deze zoekfunctie kan soms veel zoekresultaten opleveren. Daarom wordt hier [paginering](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/master/features/paginering.feature) toegepast.
+
+Op basis van de zoekresultaten kan een adres worden opgehaald met endpoint /adressen?zoekresultaatIdentificatie={zoekresultaatIdentificatie}, waarbij {zoekresultaatIdentificatie} moet worden vervangen een waarde voor identificatie in de zoekresultaten van /adressen/zoek.
```

Comment:
moet vervangen worden een waarde = moet vervangen worden **met** een waarde

Reply:
ik maak ervan: "waarbij {zoekresultaatIdentificatie} **vervangen moet worden door** een waarde voor identificatie in de zoekresultaten van /adressen/zoek."